src/components/inputs/Checkbox/Checkbox.module.scss

```
@@ -0,0 +1,129 @@
+.react-aria-Checkbox {
```

I think it’s bit confusing to use aria base class name as if it were an override, while it’s in fact hashed to module and the actual `react-aria-Checkbox` is overridden by the className prop.
```suggestion
.labelWrapper {
```